### ISSUE TITLE
Ambiance theme

### ISSUE THREAD

**Issue by yura-pakhuchiy** [seleznev/firefox-extension-htitle#75]
Default mode does not work with (ubuntu default) Ambiance theme in Ubuntu 14.04 in Gnome Shell 3.10.4. Works fine with other themes (eg. Adwaita). If I enable legacy mode it starts working fine with Ambiance as well.

However I'm scared by warning "Please do not use the legacy mode with GNOME Shell!". Why I should not? What side effects it will have? I would like to use your extension, but unfortunately it works only in legacy mode for me.

**Comment by seleznev** [comment]
> What side effects it will have?

Nothing critical, like #8, #10.

> Default mode does not work with (ubuntu default) Ambiance theme in Ubuntu 14.04 in Gnome Shell 3.10.4.

You can check that HTitle works correct by following steps:

1. Disable HTitle on about:addons page (or remove it).
2. Restart Firefox.
3. Open Terminal and type following command: `xprop -f _GTK_HIDE_TITLEBAR_WHEN_MAXIMIZED 32c -set _GTK_HIDE_TITLEBAR_WHEN_MAXIMIZED 1`
4. After press `Enter`, click on the Firefox window.
5. Try to maximize window (title bar should be hidden).
6. Repeat 1-5 steps with Adwaita theme.

If it works with Adwaita, but not with Ambiance, then it's feature of Ambiance theme.

**Comment by yura-pakhuchiy** [comment]
Yes, I followed steps above and it works with Adwaita, but not with Ambiance.

Is it possible to fix this "feature" of Ambiance? Maybe there is some flag which I just need to turn off? Or this will require not trivial theme rewrite?

**Comment by seleznev** [comment]
> Is it possible to fix this "feature" of Ambiance?

I have no idea how to do it, sorry. :(
